lib/model/CHierarchicalResultsNormalizer.cc

```diff
@@ -261,6 +261,7 @@ CHierarchicalResultsNormalizer::fromJsonStream(std::istream& inputStream) {
                 return E_Ok;
             }
         }
+        LOG_DEBUG(<< "name: " << traverser.name());
```

Comment:
Is this really going to be useful in production?

Reply:
No, removed